camus

```diff
@@ -11,3 +11,9 @@ cwd=$(perl -e 'use Cwd "abs_path";print abs_path(shift)' $0/..)
 
 CLASSPATH="$(echo ${cwd}/target/camus-*-SNAPSHOT-shaded.jar):$(hadoop classpath)"
 exec /usr/bin/java -cp $CLASSPATH com.linkedin.camus.etl.kafka.CamusJob "$@"
```

Comment:
@tundeaoni you need to remove the exec here, otherwise the remaining script will never run